torch/csrc/jit/ir.cpp

```diff
@@ -250,20 +239,7 @@ std::ostream& printNode(std::ostream & out, const Node * n, std::vector<const No
       printAttributes(out,n);
     }
   IR_END()
-  out << "(" << n->inputs() << "), uses = [";
```

Comment:
(This comment makes no requests for this PR.) I absolutely agree that, as you noted, the uses information is basically useless out of the box. However, if I ever am in the situation where I need to debug corrupt uses information (perhaps I found out about it because lint failed), the first piece of code I am going to write is printer for uses info (and probably a printer for the uniques of nodes). I don't think it should be printed by default, but I do think we should have a debug print mode for this situation. OTOH, with lint we should catch these bugs quite quickly, so I'm OK with letting the person who has to actually debug such a thing write the code.

Reply:
Yeah, I am fine with a print uses flag for debugging these issues that is off by default. But it was getting hard to read when uses would refer to nodes whose id was not being printed, and this was confusing people.